structs-benchmark/src/main/java/de/hanno/struct/benchmark/de/hanno/struct/benchmark/kotlin/KotlinBenchmarks.kt

```diff
@@ -1,57 +1,55 @@
-package de.hanno.struct.benchmark.de.hanno.struct.benchmark.kotlin
-
-import de.hanno.struct.StructArray
-import de.hanno.struct.benchmark.SimpleMutableStruct
-import de.hanno.struct.benchmark.SimpleStruct
-import de.hanno.struct.benchmark.StructBenchmark
-import de.hanno.struct.benchmark.StructBenchmark.LIST_SIZE
-import de.hanno.struct.forEach
-import org.openjdk.jmh.annotations.Benchmark
-import org.openjdk.jmh.annotations.Fork
-import org.openjdk.jmh.annotations.Measurement
-import org.openjdk.jmh.annotations.Scope
-import org.openjdk.jmh.annotations.State
-import org.openjdk.jmh.annotations.Warmup
-import org.openjdk.jmh.infra.Blackhole
-
-@State(Scope.Thread)
-open class IterateAndMutateStructArrayState {
-    @JvmField val structArray = StructArray(LIST_SIZE) { SimpleMutableStruct() }
-}
-
-@State(Scope.Thread)
-open class IterateAndMutateStructArrayIndexedState {
-    @JvmField val structArray = StructArray(LIST_SIZE) { SimpleMutableStruct() }
-}
-
-@State(Scope.Thread)
-open class IterateStructState {
-    @JvmField val structArray = StructArray(LIST_SIZE) { SimpleStruct() }
-}
-
-fun iterateAndMutateStructArray(hole: Blackhole, state: IterateAndMutateStructArrayState) {
-    state.structArray.forEach(false) { struct: SimpleMutableStruct ->
-        struct.a = struct.a + 1
-        struct.b = struct.b + 2
-        struct.c = struct.c + 3
-        hole.consume(struct)
-    }
-}
-
-fun iterateAndMutateStructArrayIndexed(hole: Blackhole, state: IterateAndMutateStructArrayIndexedState) {
-    for (i in 0 until state.structArray.size) {
-        val struct = state.structArray.getAtIndex(i)
-        struct.a = struct.a + 1
-        struct.b = struct.b + 2
-        struct.c = struct.c + 3
-        hole.consume(struct)
-    }
-}
-
-fun iterateStruct(hole: Blackhole, state: IterateStructState) {
-    state.structArray.forEach(false) { struct: SimpleStruct ->
-        hole.consume(struct.a)
-        hole.consume(struct.b)
-        hole.consume(struct.c)
-    }
+package de.hanno.struct.benchmark.de.hanno.struct.benchmark.kotlin
+
+import de.hanno.struct.StructArray
+import de.hanno.struct.benchmark.SimpleMutableStruct
+import de.hanno.struct.benchmark.SimpleStruct
+import de.hanno.struct.forEach
+import org.openjdk.jmh.annotations.Scope
+import org.openjdk.jmh.annotations.State
+import org.openjdk.jmh.infra.Blackhole
+
+
+@State(Scope.Thread)
+open class IterateAndMutateStructArrayState {
+    @JvmField
+    val structArray = StructArray(LIST_SIZE) { SimpleMutableStruct() }
```

Comment:
Import for LIST_SIZE seems missing?

Reply:
Whoops I think the IDE might've removed it for some reason. I was getting some weird "Packages cannot be imported" errors sometimes in the IDE for some bizarre reason, so that might be related.